docs/docs/configure-task-handling.md

```diff
@@ -0,0 +1,18 @@
+---
+Configuring Task Handling
+---
+
+You can configure Marathon's actions on unreachable tasks. The `unreachableStrategy` parameter of your app definition allows you to configure this in two ways: by defining when a new task instance should be launched, and by defining when a task instance should be expunged.
+
+```
```

Comment:
No, we have defaults for both ( https://phabricator.mesosphere.com/D243#88458d0e): 5 and 10 minutes.